proto/buf/alpha/registry/v1alpha1/sync.proto

```diff
@@ -82,3 +82,11 @@ message SyncGitCommitResponse {
   // SyncPoint is the latest syncpoint for the SyncGitCommit request.
   GitSyncPoint sync_point = 1;
 }
+
+// Branch holds branch information when pushed as a sync git point.
+message Branch {
+  // Name of the branch.
+  string name = 1;
+  // Is this branch the default branch in the repo.
+  bool is_repo_default = 2;
```

Comment:
This feels like it shouldn't be in the API

Reply:
I'm not a fan of this either, but the BSR has no way of knowing which one is the default repo branch to mark it as so, that's only known by the CLI when navigating the git repo locally. Any suggestions? Another option is to make it configurable in the BSR, but that would open the possibility of having a different default branch in git vs BSR, which doesn't sound attractive imo.

Reply:
I think I'd need to understand what the behavior here is supposed to be a bit better - are we no longer having the concept of a single `main` branch on the BSR, instead letting the user name this as they please? Regardless, making this a per-RPC option seems bad and open to error, this should be a setting somewhere.